packages/desktop-gui/cypress/fixtures/config.json

```diff
@@ -6,23 +6,23 @@
     {
       "name": "chrome",
       "displayName": "Chrome",
-      "family": "chrome",
+      "family": "chromium",
       "version": "78.0.3904.108",
       "path": "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
       "majorVersion": "78"
     },
     {
       "name": "chromium",
       "displayName": "Chromium",
-      "family": "chrome",
+      "family": "chromium",
       "version": "74.0.3729.0",
       "path": "/Applications/Chromium.app/Contents/MacOS/Chromium",
       "majorVersion": "74"
     },
     {
-      "name": "canary",
+      "name": "chrome",
       "displayName": "Canary",
-      "family": "chrome",
+      "family": "chromium",
```

Comment:
shouldn't the edge browsers have family: chromium instead of chrome?

Reply:
good catch

Reply:
wow, exactly whatjennifer said